appctl/appctl.go

```diff
@@ -17,10 +17,27 @@ import (
 	"github.com/openstack-k8s-operators/dataplane-node-exporter/log"
 )
 
-func Call(method string, args ...string) string {
+type appctlDaemon string
+
+const (
+	ovsVswitchd   appctlDaemon = "ovs-vswitchd"
+	ovnController appctlDaemon = "ovn-controller"
+)
+
+func Call(daemon appctlDaemon, method string, args ...string) string {
```

Comment:
Could you change this function to be private? (`s/Call/call/`) and add two small wrappers that will be invoked in other parts of the code:

```go
func OvsVSwitchd(method string, args ...string) string {
         return call(ovsVSwitchd, method, args...)
}

func OvnController(method string, args ...string) string {
         return call(ovnController, method, args...)
}
```

And then update the callers to use `appctl.OvsVSwitchd(...)` instead of `appctl.Call("ovs-vswitchd", ....)`. For your ovn-controller collecter, you will use `appctl.OvnController(...)` instead.